hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -424,6 +424,7 @@ message ContainerReplicaHistoryProto {
     required int64 lastSeenTime = 3;
     required int64 bcsId = 4;
     optional string state = 5;
+    optional int64 dataChecksum = 6;
```

Comment:
Adding this new field in proto, will there be any decoding issues for existing data in `replica_history` table in upgrade case when try to load ? Has this been tested ?

Reply:
I added this as an `optional` field, so that is saving us from some problems regarding compatibility (similarly how it was done in #4443). But I'm not sure if we should handle the case when there is no `dataChecksum` and we call this https://github.com/apache/ozone/blob/2874c8707c4790e509beb033a830a50e7c321457/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ContainerReplicaHistory.java#L96-L101
If we should, than the `state` field is also missing this and we should do it for that as well.